src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -933,10 +933,10 @@ private void FastKill()
             UnobservedExceptionsHandlerClass.ResetUnobservedExceptionHandler();
 
             SafeExecute(() => this.SystemStatus = SystemStatus.Terminated);
-            SafeExecute(LogManager.Close);
             SafeExecute(() => AppDomain.CurrentDomain.UnhandledException -= this.DomainUnobservedExceptionHandler);
             SafeExecute(() => this.assemblyProcessor?.Dispose());
             SafeExecute(() => (this.Services as IDisposable)?.Dispose());
+            SafeExecute(LogManager.Close);
```

Comment:
Not that it changes in this PR, but I believe this hinders our ability to host multiple silos per AppDomain. Of course, the larger issue is the global `LogManager`, but perhaps we could just flush logs here and keep it open?